recipe/meta.yaml

```diff
@@ -107,3 +106,4 @@ extra:
     - m3vaz
     - mmccarty
     - vyasr
+    - vzhurba01
```

Comment:
I am granting maintainer access to @vzhurba01 from the CUDA Python team, in case I am occupied and cannot attend to feedstock issues.